subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -269,7 +276,7 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
         given:
         buildFile << """
             plugins {
-                id "org.xtext.xtend" version "1.0.5"
+                id "org.xtext.xtend" version "1.0.17"
```

Comment:
The latest documentation still refers to that version of the xtend lib... but it also refers to 1.0.0 of the plugin.  Should I go ahead and update it? 

http://xtext.github.io/xtext-gradle-plugin/xtend.html